### ISSUE TITLE
A constant as the array size in a function declaration breaks highlighting of following elements

### ISSUE THREAD

**Issue by miguel164** [davidhq/SublimeEthereum#63]
Hi, it's me again ^^'

Here you can see what I mean:

![Screenshot at 2022-03-31_19:00:59](https://user-images.githubusercontent.com/15159199/161110297-1f1f3f73-aca6-4803-aef5-57653157cd0a.png)

Here is the code:

```
contract Foo {

    uint256 public constant MY_CONST = 10;

    function testWithoutConstant(uint256[10] calldata _param) public pure {
        return _param[0];
    }

    function testWithConstant(uint256[MY_CONST] calldata _param) public pure {
        return _param[0];
    }
}
```

Regards.

**Comment by davidhq** [comment]
Tnx ! I'm vaguely aware that this was potentially left unfinished... will check soon

**Comment by davidhq** [comment]
Fixed in `v1.3.7`
